SECURITY.md

```diff
@@ -0,0 +1,69 @@
+# X11Libre Security Policy
+
+##  Reporting Vulnerabilities
+
+The X11Libre project takes security seriously. If you discover any vulnerabilities, please report them responsibly.
+
+### How to Report a Security Vulnerability
+
+Send a detailed email to one or more of the following contacts:
+- info@metux.net
+- legendarydood@gmail.com
+
+Include the following information:
+
+1. **Vulnerability description**
+   - What did you observe, and why is it a concern?
+
+2. **Reproduction steps**
+   - Clear, step-by-step instructions
+   - Include specific configurations or inputs required
+
+3. **System and environment details**
+   - OS version
+   - X11Libre version or commit hash
+   - Display manager, drivers, or hardware specifics
+
+4. **Supporting data**
+   - Logs (in plain text)
+   - Core dumps (if available and safe to share)
+
+5. **Impact analysis (if known)**
+   - Potential for remote or local exploitation
+   - Possible consequences (e.g. data exposure, privilege escalation, denial-of-service)
+
+Please allow us ample time to validate and patch the issue before disclosing it publicly.
+
+Feel free to privately message staff over our official Matrix or Telegram if the issue is of extreme merit and needs an immediate solution. 
+
+##  Supported Versions
+
+| Version         | Status                    |
+| --------------- | ------------------------- |
+| `master` branch |  Supported and maintained |
+| Older tags      |  No longer supported      |
```

Comment:
What does that mean for distros that rely on X11Libre? As an example, the distros that compiled and used https://github.com/X11Libre/xserver/releases/tag/xlibre-xserver-25.0.0.3 won't receive security updates?

Reply:
As long as we don't really have many existing releases to care for, that whole list doesn't make much sense now.
all releases are getting security fixes.
Let's revise that again, when we have to make a cut and drop some.